src/pages/Ecommerce/Ecommerce.jsx

```diff
@@ -1,10 +1,208 @@
 import React from 'react';
 import Header from '../../components/Header';
+import hamburger from '../../assets/ecommerce-img/icon-menu.svg';
+import close from '../../assets/ecommerce-img/icon-close.svg';
+import cart from '../../assets/ecommerce-img/icon-cart.svg';
+import avatar from '../../assets/ecommerce-img/image-avatar.png';
+import logo from '../../assets/ecommerce-img/logo.svg';
+import { useState } from 'react';
+import './Ecommerce.css';
+
+import minus from '../../assets/ecommerce-img/icon-minus.svg';
+import plus from '../../assets/ecommerce-img/icon-plus.svg';
+import DesktopGallery from './DesktopGallery/DesktopGallery';
+import Cart from './Cart';
+import GalleryComponent from './DesktopGallery/GalleryComponent';
 
 function Ecommerce() {
+  const [nav, showNav] = useState(false);
+  const [toggle, setToggle] = useState(false);
+  const [index] = useState(0);
+  const [items, setItems] = useState(0);
+  let [quantity, setQuantity] = useState('');
+  const [cartModal, setCartModal] = useState(false);
+
+  const openNav = () => {
+    showNav(!nav);
+    toggle ? setToggle(false) : setToggle(true);
+  };
+
+  const increment = () => {
+    quantity++;
```

Comment:
Don't increment the state variable directly, that's why you have the setter.
Just do `setQuantity(quantity + 1)`